include/wx/image.h

```diff
@@ -628,6 +651,46 @@ class WXDLLIMPEXP_CORE wxImage: public wxObject
     // modified versions of this image.
     wxImage MakeEmptyClone(int flags = Clone_SameOrientation) const;
 
+    // Helper function used internally by wxImage class only.
+    template <typename T>
+    void ApplyToAllPixels(void (wxImage::*func)(unsigned char *, T), T value);
+
+    // Makes an RGB value disabled.
+    void DoMakeDisabled(unsigned char *rgb, unsigned char brightness);
+
+    // red, green and blue are weight doubles, they are used internally by
+    // DoMakeGrey() only.
+    class WeightValue
```

Comment:
If we don't need `DoXXX()` to be member functions, they, and this class, should be moved to `image.cpp`.

Reply:
Absolutely, but in an unnamed namespace or each one just before its caller function?